### ISSUE TITLE
[userTypeFix] Code Coverage: Create tests for src/screens/OrgList/OrgList.tsx

### ISSUE THREAD

**Issue by varshith257** [PalisadoesFoundation/talawa-admin#1771]
We will need unittests done for all methods, classes and functions found in this file.

Any widgets, components, modals referenced in this file must also have unittests done

PR Acceptance Criteria

1. When complete this file must show at least 100% coverage when merged into the code base.
2. The PR will show a report for the code coverage for the file you have added. You can use that as a guide

**Comment by varshith257** [comment]
@adi790uu  Get it assigned and make a PR once it is done

**Comment by adi790uu** [comment]
Yeah I would like to work on this.

**Comment by palisadoes** [comment]
We are tracking this issue with one of our projects, and we'd like to track its progress especially as the GSoC announcement dates draw near.

1. When you create your PR, please link it back to this project:
    1. https://github.com/orgs/PalisadoesFoundation/projects/24/views/1
1. The process is simple. It can be found here:
    1. https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#manually-linking-a-pull-request-to-an-issue-using-the-pull-request-sidebar

**Comment by aarishshahmohsin** [comment]
Can I be assigned this issue?

**Comment by palisadoes** [comment]
At 93% coverage. We currently have other priorities
